jdaviz/configs/imviz/tests/test_simple_aper_phot.py

```diff
@@ -249,6 +250,13 @@ def test_annulus_background(imviz_helper):
     phot_plugin.bg_annulus_width = 5
     assert_allclose(phot_plugin.background_value, 4.894003242594493)
 
+    # Move the last created Subset (ellipse) and make sure background updates
+    imviz_helper.app.session.edit_subset_mode.mode = ReplaceMode

```

Comment:
Devs, I am not familiar with the edit mode syntax, so please let me know if it is correct to test this way.

Would also be nice if you can tell me how to programmatically control which Subset to put in ReplaceMode.

Thanks!

Reply:
To programmatically set the subset in replace mode, you can do something like:

```
imviz_helper.app.session.edit_subset_mode.edit_subset = [imviz_helper.app.data_collection.subset_groups[0]]
```